modules/odc-deploying-java-applications.adoc

```diff
@@ -2,40 +2,42 @@
 [id="odc-deploying-java-applications_{context}"]
 = Upload JAR files for easy deployment of Java applications
 
-You can use the JAR files in the *Topology* view of the *Developer* perspective to deploy your Java applications. You can upload a JAR file using the following options:
+You can use the web console *Developer* perspective to upload a JAR file by using the following options:
 
 * Navigate to the *+Add* view of the *Developer* perspective, and click *Upload JAR file* in the *From Local Machine* tile. Browse and select your JAR file, or drag and drop a JAR file to deploy your application.
 
 * Navigate to the *Topology* view and use the *Upload JAR file* option, or drag and drop a JAR file to deploy your application.
 
 * Use the in-context menu in the *Topology* view, and then use the *Upload JAR file* option to upload your JAR file to deploy your application.
 
-Use the following instructions to upload a JAR file in the *Topology* view to deploy a Java application:
+.Prerequisites
+
+* The Cluster Samples Operator must be installed by a cluster administrator.
+* You have access to the {product-title} web console and are in the *Developer* perspective.
 
 .Procedure
 
-. In the *Topology* view, right-click anywhere in the *Topology* view to see the *Add to Project* menu.
+. In the *Topology* view, right-click anywhere to view the *Add to Project* menu.
 
-. Hover over the *Add to Project* menu to see the menu options, and then select the *Upload JAR file* option to see the *Upload JAR file* form . Alternatively, you can drag and drop the JAR file in the *Topology* view.
+. Hover over the *Add to Project* menu to see the menu options, and then select the *Upload JAR file* option to see the *Upload JAR file* form. Alternatively, you can drag and drop the JAR file in the *Topology* view.
 
 . In the *JAR file* field, browse for the required JAR file on your local machine and upload it. Alternatively, you can drag and drop the JAR file on the field. A toast alert is displayed at the top right if an incompatible file type is dragged and dropped on the *Topology* view. A field error is displayed if an incompatible file type is dropped on the field in the upload form.
```

Comment:
```suggestion
. In the *JAR file* field, browse for the required JAR file on your local machine and upload it. Alternatively, you can drag the JAR file on the field. A toast alert is displayed at the top right if an incompatible file type is dragged on the *Topology* view. A field error is displayed if an incompatible file type is dropped on the field in the upload form.
```
From the IMB style guideline:
drag and drop (verb): Do not use. Use "drag", which includes the action of dropping an object.